community/modules/compute/schedmd-slurm-gcp-v5-partition/README.md

```diff
@@ -32,6 +32,51 @@ The following code snippet creates a partition module with:
 For a complete example using this module, see
 [slurm-gcp-v5-cluster.yaml](../../../examples/slurm-gcp-v5-cluster.yaml).
 
+### Compute VM Zone Policies
+
+> **_WARNING:_** Lenient zone policies can lead to additional egress costs when
+> moving data between Google Cloud resources in different zones in the same
+> region, such as between filestore and other VM instances. For more information
+> on egress fees, see the [Network Pricing][networkpricing] Google Cloud
+> documentation.
+>
+> To avoid egress charges, ensure your compute nodes are created in the same
+> zone as the other resources that share data with them by setting
+> `zone_policy_deny` to all other zones in the region.
+
+The Slurm on GCP partition modules provide the option to set policies regarding
+which zone the compute VM instances will be created in through the
+`zone_policy_allow` and `zone_policy_deny` variables.
+
+As an example, see the the following module: will show preference for zones
+`us-central1-a` and `us-central1-b` while explicitly not allowing any VM
+instances to be created in `us-central1-f`
+
+```yaml
+- id: partition-with-zone-policy
+  source: community/modules/compute/schedmd-slurm-gcp-v5-partition
+  settings:
+    zone_policy_allow:
+    - us-central1-a
+    - us-central1-b
+    zone_policy_deny: [us-central1-f]
+```
+
+In this module, the following is defined:
+
+* `us-central1-a` and `us-central1-b` zones have been explicitly allowed.
+* `us-central1-f` has been explicitly denied, therefore no nodes in this
+  partition will be created in that zone.
+* Since `us-central1-c` was not included in the zone policy, it will default to
+  "Allow", which means the partition has the same liklihood of created a node in
```

Comment:
- misspelling of "liklihood"
- you use "region" when you mean "zone"
- you use "created" when you mean "creating"
- I would probably not say "2 listed under" and say instead "as zones explicitly listed..."

Reply:
Fixed